docs/user/faq.rst

```diff
@@ -1,65 +1,71 @@
 Frequently Asked Questions
 ==========================
 
-My project isn't building correctly
------------------------------------
+.. contents::
+   :local:
 
-First, you should check out the Builds tab of your project.
-That records all of the build attempts that RTD has made to build your project.
-If you see ``ImportError`` messages for custom Python modules,
-see our section on :ref:`faq:My documentation requires additional dependencies`.
+..
+  Frequently asked questions should be questions that actually got asked.
+  Formulate them as a question and an answer.
+  Consider that the answer is best as a reference to another place in the documentation.
 
-If you are still seeing errors because of C library dependencies,
-please see :ref:`faq:I get import errors on libraries that depend on C modules`.
 
+Building and publishing your project
+------------------------------------
 
-Help, my build passed but my documentation page is 404 Not Found!
------------------------------------------------------------------
 
-This often happens because you don't have an `index.html` file being generated.
-Make sure you have one of the following files:
+.. Old reference
```

Comment:
When you see this comment, it means that I've edited the subsection title.

Reply:
Good approach. The goal here is not breaking inbound links?  👍

Reply:
If the answer is "Yes", this is probably a good recommendation for the "Automatic redirect" page you are working on 👍🏼

Reply:
Thanks for that @humitos !!

Yes, I did a test yesterday and there's an invisible `<span id="old_ref">` inserted which makes `page.html#anchor` retained, as well as `:ref:`. I didn't test with Intersphinx, but I'm 99.9% sure that'll work as well.